### ISSUE TITLE
Adguard is still running even after disabling it on particualr website only

### ISSUE THREAD

**Issue by Chetan7575** [AdguardTeam/AdguardBrowserExtension#2857]
### Please answer the following questions for yourself before submitting an issue

- [x] Filters were updated before reproducing an issue
- [X] I checked the [knowledge base](https://adguard.com/kb/) and found no answer
- [X] I checked to make sure that this issue has not already been filed

### AdGuard Extension version

4.3.53

### Browser version

Chrome Version 126.0.6478.127

### OS version

Win10 - 64Bit

### Ad Blocking

AdGuard Base filter, EasyList

### Annoyances

AdGuard Annoyances filter, AdGuard Popups filter, Adblock Warning Removal List, Fanboy's Annoyances

### Security

Online Malicious URL Blocklist, Phishing URL Blocklist, Scam Blocklist by DurableNapkin, uBlock Origin – Badware risks

### Other

Filter unblocking search ads and self-promotion

### Language-specific

AdGuard Chinese filter, AdGuard French filter, AdGuard German filter, AdGuard Japanese filter, AdGuard Russian filter, AdGuard Spanish/Portuguese filter, AdGuard Turkish filter, Bulgarian list, Dandelion Sprout's Nordic Filters, EasyList Italy, EasyList Thailand, IndianList

### Issue Details

When using the admin panel of Taboola, a specific URL fails to load if AdGuard's language filters are enabled, even when AdGuard is disabled for the website. This issue breaks the website and prevents the report from being generated.

AdGuard Status:
AdGuard is disabled for the website.
Language-specific filters in AdGuard are enabled.
Problem:
With language-specific filters enabled, the URL request gets blocked before it executes, causing the website to break and the report generation to fail.
When language filters are turned off, the URL loads successfully, and the network request executes correctly.

Steps to Reproduce:
You can't reproduce it, but I'm explaining.
Disable AdGuard for the Taboola admin panel website.
Ensure language filters in AdGuard are enabled.
In the network panel URL request gets blocked, breaking the website and preventing report generation.
Disable the language filters in AdGuard.
Attempt to load the URL again and observe that it loads successfully.

Expected Behavior:
The URL should load successfully, and the report should generate correctly even if the language filters are enabled, given that AdGuard is disabled for the website.

URL: https://backstage.taboola.com/backstage/resources/components/requirejs/require.js
Username: @Chetan7575

You can also refer to this issue on: https://github.com/AdguardTeam/AdguardFilters/issues/182230

### Expected Behavior

The URL in the network tab should load successfully, and the report should generate correctly even if the language filters are enabled, given that AdGuard is disabled for the website.

### Actual Behavior

The URL in the network tab doesn't load successfully, and the report is not being generated even if the language filters are enabled, and the AdGuard is disabled for the website.

### Screenshots

No screenshot

### Additional Information

https://github.com/AdguardTeam/AdguardFilters/issues/182230

From the above ticket I created earlier, I found that in language-specific filters - Easylist Thailand causes the taboola request block in the network tab. So I turned off this filter and the URL in the network loaded successfully. But it should not happen cause adguard is disabled on website.

**Comment by alexx7311** [comment]
@Chetan7575 Hello! Please do the following:
1. Add taboola.com to the Allowlist
2. Open the filtering log (AG extension icon > Open filtering log) on the page you encounter the problem on
3. Refresh the page for requests to load
4. Type `Thailand` in the search field of the filtering log
5. Take a screenshot of the filtering log

**Comment by Chetan7575** [comment]
Now close it, men, 😪
I disabled language-specific filters and it's working fine

4. Type Thailand in the search field of the filtering log
as you said, I enabled all filters again and then tried what you said, and there was no result when I searched Thailand.

And the issue
now, out of 5 refreshes, it happens in 2, which means sometimes the report doesn't show and sometimes it shows when language filters are enabled. I checked the network requests, and there are blocked requests, but in the Adguard filtering log, there are no blocked requests.
